vendor/init.bat

```diff
@@ -299,17 +315,22 @@ if exist "%CMDER_ROOT%\vendor\git-for-windows" (
 )
 
 :SPECIFIED_GIT
-%print_debug% init.bat "Using /GIT_INSTALL_ROOT..."
+%print_debug% init.bat "Using specified GIT_INSTALL_ROOT=%GIT_INSTALL_ROOT%...."
 goto :CONFIGURE_GIT
 
 :FOUND_GIT
-%print_debug% init.bat "Using found Git '%GIT_VERSION_USER%' from '%GIT_INSTALL_ROOT%..."
+%print_debug% init.bat "Using found Git '%GIT_VERSION_USER%' from 'v%GIT_INSTALL_ROOT%..."
```

Comment:
@daxgames Can you please explain why `v` was prefixed? I'm confused

Reply:
No idea.

Reply:
Fixed